src/Client/Pages/Authentication/ForgotPassword.razor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FSH.BlazorWebAssembly.Client.Infrastructure.ApiClient;
+using FSH.BlazorWebAssembly.Client.Shared;
+
+
+namespace FSH.BlazorWebAssembly.Client.Pages.Authentication;
+public partial class ForgotPassword
+{
+    public ForgotPassword()
+    {
+
+    }
+
+    private ForgotPasswordRequest _forgotPasswordRequest = new ForgotPasswordRequest();
+    public bool BusySubmitting { get; set; } = false;
+
+    private CustomValidation? _customValidation;
+
+    [Inject]
+    public IIdentityClient _identityClient { get; set; } = default!;
+
+    public string _tenant { get; set; } = "root";
```

Comment:
Should probably come from a constant somewhere... I know it's already on the server, but probably not on the client yet... Anyway, just making a mental note... no reason to stop this pr ;-)